pkg/cmd/release/list/http.go

```diff
@@ -2,13 +2,25 @@ package list
 
 import (
 	"net/http"
+	"reflect"
+	"strings"
 	"time"
 
 	"github.com/cli/cli/v2/api"
 	"github.com/cli/cli/v2/internal/ghrepo"
 	"github.com/shurcooL/githubv4"
 )
 
+var ReleaseFields = []string{
```

Comment:
Two things here:
1. This does not need to be exported
```suggestion
var releaseFields = []string{
```

2. In other `release` commands we already support the `--json` export and we should try to match the [fields](https://github.com/cli/cli/blob/b8a1ff6e70dc9bbe12a8ffdb5644889de6be48a4/pkg/cmd/release/shared/fetch.go#L23) that are available there as best we can. We are using the GraphQL endpoint here so not all the fields will be available but the ones that are would be nice to include. One exception I would say are the assets. Including the assets would dramatically increase the data retrieved from the `release list` command and could have detrimental UX implications.